static/src/js/actions/__tests__/focusedCollection.test.js

```diff
@@ -162,8 +162,8 @@ describe('getFocusedCollection', () => {
         expect(getSearchGranulesMock).toHaveBeenCalledTimes(1)
       })
 
-      describe('when the requested collection is cwic and a polygon search is active', () => {
-        test('should toggle the polygon warning, update the focusedCollection and call getSearchGranules', async () => {
+      describe('Testing focusedColection does not retrieve a non existant gibs tag', () => {
```

Comment:
This describe and test block didn't change in a significant way, revert back to the original text, it matches the naming convention of all our tests.